src/main/resources/db/migration/postgresql/V1_10__2055_Topic_Column_On_Workstack_Team_Queue.sql

```diff
@@ -0,0 +1,16 @@
+DROP VIEW IF EXISTS stage_data;
+
+CREATE OR REPLACE VIEW stage_data AS
```

Comment:
Do we need to drop the views, the create or replace view will do this for you? views act as virtual tables and thus from my knowledge don't hold info.